docs/i18n.md

```diff
@@ -0,0 +1,103 @@
+# i18n
+
+## Context
+
+Internationalization on Force was introduced as a hackathon project
```

Comment:
Is there a tool that splits the lines like this? It doesn't make much sense for documentation.

Reply:
We were working on Notion and I guess I messed it up and I copied it from there, thanks for raising this!!